sentry/src/main/java/io/sentry/util/HintUtils.java

```diff
@@ -16,6 +19,24 @@ public final class HintUtils {
 
   private HintUtils() {}
 
+  @ApiStatus.Internal
+  public static void setIsFromHybridSdk(Hint hint, String sdkName) {
+    if (sdkName.startsWith(SENTRY_REACT_NATIVE_SDK_NAME)
+        || sdkName.startsWith(SENTRY_DART_SDK_NAME)) {
```

Comment:
L: Other Hybrid SDKs would have the same issue I suspect, but we can expand this later.

Reply:
I've changed it a bit because we can consider all the SDKs based on JS, Dotnet, or Dart them hybrid then the check can be more universal.